client/server-connection.hpp

```diff
@@ -61,6 +62,11 @@ namespace tec {
 				return this->average_ping;
 			}
 
+			ping_time_t GetEstimatedDelay() {
+				ping_time_t since_recv = std::chrono::duration_cast<std::chrono::milliseconds>(
+					std::chrono::high_resolution_clock::now() - recv_time).count();
+				return (this->stats.estimated_server_time - this->stats.last_state_time) + since_recv;
```

Comment:
This method is used for displaying the state delay, it's not reliable and currently only for stats display